cps/tasks/metadata_extract.py

```diff
@@ -110,7 +110,7 @@ def _update_metadata(self, requested_urls):
         requested_urls = {url: requested_urls[url] for url in requested_urls.keys() if "shorts" not in url and url not in failed_urls}
 
     def _calculate_views_per_day(self, requested_urls, conn):
-        now = datetime.now()
+        now = datetime.now(timezone.utc)
```

Comment:
The advice of using `time.monotonic()` does *NOT* apply here, because later time_uploaded is subtracted from it to compute a time difference.

Both `time.monotonic()` and `time.time()` are floats, but they are not based on t he same epoch:

```pycon
>>> import time
>>> time.monotonic()
180162.46153954
>>> time.time()
1720317895.1978724
```

Thus, subtracting a monotonic clock from a system clock is a serious bug.